src/CommunityToolkit.Maui/Converters/StringToListConverter.shared.cs

```diff
@@ -8,15 +7,61 @@ namespace CommunityToolkit.Maui.Converters;
 /// </summary>
 public class StringToListConverter : BaseConverterOneWay<string?, IEnumerable<string>>
 {
+	string separator = " ";
+	IList<string> separators = Array.Empty<string>();
+
 	/// <summary>
-	/// The string that delimits the substrings in this string.
+	/// The string that delimits the substrings in this string
+	/// This value is superseded by the ConverterParameter, if provided
+	/// This value is also superseded by <see cref="Separators"/>, if provided
+	/// If ConverterParameter is null and <see cref="Separators"/> is not assigned, this property will be used to delimit the substrings
 	/// </summary>
-	public string Separator { get; set; } = string.Empty;
+	public string Separator
+	{
+		get => separator;
+		set
+		{
+			if (value is null)
+			{
+				throw new ArgumentNullException(nameof(value));
+			}
+
+			if (string.IsNullOrEmpty(value))
+			{
+				throw new ArgumentException("An empty string is not a valid separator", nameof(value));
+			}
+
+			separator = value;
+		}
+	}
 
 	/// <summary>
-	/// The strings that delimits the substrings in this string.
+	/// The strings that delimits the substrings in this string
```

Comment:
Minor but non plural delimit.

```suggestion
	/// The strings that delimit the substrings in this string
```